### ISSUE TITLE
How to specify CaseStrategy for MapMapper in SqlObject Extension?

### ISSUE THREAD

**Issue by brainbytes42** [jdbi/jdbi#2665]
I have a DAO-interface, which is driven by the SQL Objects-Extension.

For one Method I have this annotations: 
```java
@SqlQuery( ... )
@RegisterRowMapper(MapMapper.class)
List<Map<String, Object>> myMethod( ... );
```

I'd like to have the RowMapper to NOT change the column name's cases, i.e. apply `CaseStrategy.NOP`. 
But how do I make the MapMapper do that?

Thanks!

**Comment by lutzgh** [comment]
You can use `MapMappers::setCaseChange(UnaryOperator<String> caseChange)` do do this.

https://github.com/jdbi/jdbi/blob/7033ec5b04a0832d843f2310de51b97aaaf58e88/core/src/main/java/org/jdbi/v3/core/mapper/MapMappers.java#L48

**Comment by brainbytes42** [comment]
Thanks for the quick reply.

My problem is: where would I put this setting? Maybe I'm missing something obvious, but as I have only my on-demand sql-object DAO-interface with its annotation, all I have is the MapMapper.class as reference to the class, but no object to change the setting...?

**Comment by stevenschlansker** [comment]
Good point - this looks like it might be an oversight. You can always call `handle.registerRowMapper(myConfiguredMapMapper)` but this should also be possible to do in annotation-driven configuration.
